### PR TITLE
Add a readme and change the employees file

### PR DESCRIPTION
**Pull request by SteevenR**
This commit adds a README file with some instructions for practice
It also change a little bit the employees file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+# Introduction
+This project is here to see how wa could improve a code base.
```
Comment (VV01K) [inline README.md]:
```suggestion
This project is here to see how we could improve a code base.
```

File: birthday-greetings/employees.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿first_name, last_name, date_of_birth, email
 Benoit, Gantaume, 26/12/1979, benoit@artisandeveloppeur.fr
 John, Doe, 12/11/1900, john.doe@gmail.com
-Bill, Dilbert, 11/12/1900, john.doe@gmail.com
 Eight, March, 08/03/1990, john.doe@gmail.com
 Emile, Poli, , emile.poli@gmail.com
-No, Email
\ No newline at end of file
+No, Email
+Bill, Dilbert, 11/12/1900, john.doe@gmail.com
```
Comment (VV01K) [inline birthday-greetings/employees.txt]:
```suggestion
Bill, Dilbert, 11/13/1900, john.doe@gmail.com
```

Comment (SteevenR) [reply birthday-greetings/employees.txt]: Du coup lui on fait quoi on le laisse exprès : Write a test to verify that Bill will receive his Greetings?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Launch the program
+- Make the error more explicit
+- Run the tests
+- The 8 of march the test passed. Why the test fails today (if we are not the 8 of march) ?
+- Write a more explicit test ; want the program to send (not a real mail) an email to Benoit
+- What other tests could you write ?
+- Make sure that the whole employees file is parsed
+- Write a test to verify that Bill will receive his Greetings
+- How can you improve the SendGreetings method in term of lisibility ?
+- What responsibility can be moved out ? To Whom ?
```
Comment (VV01K) [inline README.md]:
```suggestion
# Instructions 
Here are some instructions to follow if you want to practice.

## Steps
- Launch the program and understand what it supposed to do
- Find and fix the problem
- Make the error more explicit
- Run the tests
- The 8 of march the test passed. Why the test fails today ?
- Write a more explicit test ; we want the program to send (not a real mail) an email to Benoit
- What other tests could you write ?
- Make sure that the whole employees file is parsed
- Write a test to verify that Bill will receive his Greetings
- How can you improve the SendGreetings method in term of readability ?
- What responsibility can be moved out ? To Whom ?
```

File: birthday-greetings/employees.txt [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿first_name, last_name, date_of_birth, email
 Benoit, Gantaume, 26/12/1979, benoit@artisandeveloppeur.fr
 John, Doe, 12/11/1900, john.doe@gmail.com
-Bill, Dilbert, 11/12/1900, john.doe@gmail.com
 Eight, March, 08/03/1990, john.doe@gmail.com
 Emile, Poli, , emile.poli@gmail.com
-No, Email
\ No newline at end of file
+No, Email
+Bill, Dilbert, 11/13/1900, john.doe@gmail.com
```
Comment (VV01K) [inline birthday-greetings/employees.txt]:
```suggestion
Bill, Dilbert, 11/12/1900, john.doe@gmail.com
```

File: birthday-greetings/employees.txt [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ﻿first_name, last_name, date_of_birth, email
 Benoit, Gantaume, 26/12/1979, benoit@artisandeveloppeur.fr
 John, Doe, 12/11/1900, john.doe@gmail.com
```
Comment (VV01K) [inline birthday-greetings/employees.txt:3]:
```suggestion
John, Doe, 12/13/1900, john.doe@gmail.com
```

**Review by VV01K — APPROVED** [review]

**Pull request merged**